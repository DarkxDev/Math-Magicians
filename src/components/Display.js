const Display = () => {
  return (
    <div className="display-row">
        <input type="text" className="display" readOnly/>
    </div>
  );
};

export default Display;