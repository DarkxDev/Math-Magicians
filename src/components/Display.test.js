import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import Display from './Display';

test('Renders correctly', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <Display display="test display" />
    </MemoryRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
