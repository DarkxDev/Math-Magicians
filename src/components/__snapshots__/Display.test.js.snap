// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="display-row"
>
  <input
    className="display"
    readOnly={true}
    type="text"
    value="test display"
  />
</div>
`;
